pkgs/top-level/all-packages.nix

```diff
@@ -12564,6 +12564,10 @@ with pkgs;
 
   redir = callPackage ../tools/networking/redir { };
 
+  redlib = callPackage ../by-name/re/redlib/package.nix {
+    inherit (darwin.apple_sdk.frameworks) Security;
```

Comment:
Why do we need this inherit?

Reply:
But `darwin` is in its args. I assume in `by-name` we should avoid using arbitrary callPackage function when possible.

Reply:
Thank you, I didn't know about that.